src/main/scala/com/github/unchama/seichiassist/subsystems/buildcount/application/actions/LevelUpNotifier.scala

```diff
@@ -23,9 +29,18 @@ case class LevelUpNotifier[F[_], Player]()(
     import cats.implicits._
 
     val Diff(oldLevel, newLevel) = diff
-    if (newLevel eqv BuildAssistExpTable.maxLevel)
-      SendMinecraftMessage[F, Player].string(player, s"${GOLD}最大Lvに到達したよ(`･ω･´)")
-    else if (oldLevel < newLevel)
+    if (newLevel eqv BuildAssistExpTable.maxLevel) {
+      val bukkitPlayer = player.asInstanceOf[org.bukkit.entity.Player]
+      F.pure {
```

Comment:
```suggestion
      F.delay {
```

`sendMessageToEveryoneIgnoringPreference`は純粋ではありません

Reply:
これはちょっと cats / cats-effect のレイヤの知識が要りますね
18行目で `F` に対して `Applicative` しか仮定していないのですが、 `Applicative` 型クラスのインスタンスは「任意の同期的な副作用である `() => A` を内包している」という仮定は置かれていない、より一般的なものですので、 `Sync` 型クラスまで要求してやる必要があります。

Reply:
なるほど。
やってみます！